apps/a11y-tests/just.config.js

```diff
@@ -0,0 +1,6 @@
+const { preset, just } = require('@uifabric/build');
```

Comment:
This is up for debate actually. I think just-task.js might be a bit confusing if you've never seen this before (is it a config or is it something else?) It's bit of bikeshedding I realized. If you think just-task.js is better and more clear to us, I can change 'em back!

Reply:
Thinking about it I actually like just.config.js better, since it's more obviously some kind of configuration file. It was just surprising to see that. (And I think if we're changing to that name, we ought to work on updating the convention everywhere.)

Reply:
yeah I'm glad you agree :)